lib/srv/db/mysql/autousers.go

```diff
@@ -108,6 +108,11 @@ func (c *clientConn) maxRoleLength() int {
 	return mysqlMaxUsernameLength
 }
 
+// Close calls conn.Quit to send COM_QUIT then close the conn.
+func (c *clientConn) Close() error {
+	return trace.Wrap(c.Conn.Quit())
```

Comment:
`clientConn` will use `Conn.Quit` instead of `Conn.Close`. `clientConn` will be used for testing as well.